src/db/index.js

```diff
@@ -15,35 +16,38 @@ module.exports.createUser = async (username, password, email, gitAccount, admin
         admin=${admin}`,
   );
 
-  const data = {
-    username: username,
-    gitAccount: gitAccount,
-    email: email,
-    admin: admin,
-  };
+  await bcrypt.hash(password, 10, async function (err, hash) {
+    const data = {
+      username: username,
+      password: hash,
+      gitAccount: gitAccount,
+      email: email,
+      admin: admin,
+    };
 
-  if (username === undefined || username === null || username === '') {
-    const errorMessage = `username ${username} cannot be empty`;
-    throw new Error(errorMessage);
-  }
+    if (username === undefined || username === null || username === '') {
+      const errorMessage = `username ${username} cannot be empty`;
+      throw new Error(errorMessage);
+    }
```

Comment:
Seems like your local VSCode setup is not consistent with the project prettier and format settings. Can we address?